src/printing.ts

```diff
@@ -115,10 +115,19 @@ class Printing {
         }
       );
 
-      EscPosPrinter.printBuffer(value, params).catch((e: Error) => {
-        removeListeners();
-        rej(e);
-      });
+      if (params?.target) {
```

Comment:
Can we add more checks for now to have crash on Android ?

Reply:
Alright will add the check on ios for now.